archinstall/lib/disk/device_handler.py

```diff
@@ -133,6 +133,28 @@ def get_parent_device_path(self, dev_path: Path) -> Path:
 		lsblk = get_lsblk_info(dev_path)
 		return Path(f'/dev/{lsblk.pkname}')
 
+	def get_unique_path_for_device(self, dev_path: Path) -> Optional[Path]:
+		disks_by_id = os.listdir('/dev/disk/by-id')
```

Comment:
Using `Path` instead will save some conversations
```
Path('/dev/disk/by-id').glob('*')
```
The glob can then also filter away `wwn-` and `nvme-eui.` right away

Reply:
Basically this should do 
```
paths = Path('/dev/disk/by-id').glob('*')
linked_targets = {p.resolve(): p for p in paths}

if dev_path in linked_targets:
    return linked_targets[dev_path]

```

Reply:
I'd like to keep the behavior of `wwn-*` and `nvme-eui.*` paths being prioritized. Does this code look okay or is there a better way to achieve this?

```py
paths = Path('/dev/disk/by-id').glob('*')
linked_targets = {p.resolve(): p for p in paths}
linked_wwn_targets = {p: unique_targets[p] for p in linked_targets
	if p.name.startswith('wwn-') or p.name.startswith('nvme-eui.')}

if dev_path in linked_wwn_targets:
    return linked_wwn_targets[dev_path]

if dev_path in linked_targets:
    return linked_targets[dev_path]
```